release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -89,6 +89,24 @@ For `console` and `downloads` routes, custom routes functionally is now implemen
 [id="ocp-4-8-networking"]
 === Networking
 
+[id="ocp-4-8-network-policy-host-network-ingress-controllers"]
+==== Network policy supports selecting host network Ingress Controllers
+
+When using the OpenShift SDN or OVN-Kubernetes cluster network providers, you can select traffic from Ingress Controllers in a network policy rule regardless of whether an Ingress Controller runs on the cluster network or the host network.
+In a network policy rule, the `policy-group.network.openshift.io/ingress: ""` namespace selector always matches traffic from an Ingress Controller.
+
+In earlier releases of {product-title} the following limitations existed:
+
+- A cluster using the OpenShift SDN cluster network provider could select traffic from an Ingress Controller on the host network only by applying the `policy-group.network.openshift.io/ingress: ""` label to the `default` namespace.
```

Comment:
@jboxman In earlier releases the label was 'network.openshift.io/policy-group: ingress' that could be applied to the 'default' namespace.

Reply:
@asood-rh, fixed, thanks!